README.md

```diff
@@ -19,18 +19,34 @@ cd Python-Scripts/bin
 ```
 
 ## Scripts
-* ### Approximating *pi*
 
-This script is useful to show a way to approximate the value of pi using a Monte Carlo method. It is also optimized using the `@jit` (*just-in-time*) decorator from the [numba](https://numba.pydata.org/) library.
+* ### Air Quality Script
+    - This script renders the air quality from the location of the user
+    - The location is fetched using the user's IP address
 
-* ### Meme Density
-This script tells you the amount of memes in your facebook feed
+    - This script requires a key from WAQI. 
+    - It is free and can be fetched from http://aqicn.org/data-platform/token/#/
+    #### Usage Instructions
+    ```python
+    python air-quality.py token
+    ```
+
+#### NOTE:  

```

Comment:
this is with youtube bot.py

Reply:
So the line below belongs to youtube bot.py ?